tests/Integration/SDK/Common/InMemoryStorageManagerWithSKDAutoload.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Integration\SDK\Common;
+
+use OpenTelemetry\API\Globals;
+use OpenTelemetry\API\LoggerHolder;
+use OpenTelemetry\API\Logs\LogRecord;
+use OpenTelemetry\SDK\Common\Configuration\Variables;
+use OpenTelemetry\SDK\Common\Export\InMemoryStorageManager;
+use OpenTelemetry\SDK\Logs\LoggerProvider;
+use OpenTelemetry\SDK\Metrics\MeterProvider;
+use OpenTelemetry\SDK\SdkAutoloader;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+use OpenTelemetry\Tests\TestState;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class InMemoryStorageManagerWithSKDAutoload extends TestCase
+{
+    use TestState;
+
+    protected LoggerInterface&MockObject $logger;
+
+    public function setUp(): void
+    {
+        $this->logger = $this->createMock(LoggerInterface::class);
+        LoggerHolder::set($this->logger);
+        Globals::reset();
```

Comment:
this is called in `TestState::tearDown`, so it should work with this line removed...I think :)